service/api/api.go

```diff
@@ -55,10 +55,10 @@ func (a *API) InitializeMiddleware() error {
 	if err != nil {
 		return err
 	}
-	accessLogMiddleware, err := middleware.NewAccessLog()
-	if err != nil {
-		return err
-	}
+	//accessLogMiddleware, err := middleware.NewAccessLog()
```

Comment:
I would completely remove (not just comment out). The code is in the git history so can be recovered if ever necessary.